infrastructure/testdriver/set_permission.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver set_permission method</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<script>
+async_test(t => {
+  let permission_params = {
+    descriptor: {
+      name: "ambient-light-sensor"
+    },
+    state: "granted",
+    oneRealm: true,
```

Comment:
Can you add a test with `oneRealm: false` if that does something observably different?

Reply:
Added a test that omits the param (which will default to false afaik), just to make sure it doesn't fail. I'm not entirely sure what this param does, but that seems like it should be left for non-infra tests, right?

Reply:
I also don't know what it does actually, if it doesn't have an effect that's independent of the permission in question, then it would have to be left to non-infra tests.